.circleci/config.yml

```
@@ -1,35 +1,49 @@
 version: 2.1
 
+# Checks out PyTorch and builder repo
+binary_checkout: &binary_checkout
+  name: Checkout pytorch/builder repo
```

```suggestion
  name: Checkout pytorch/pytorch and pytorch/builder
```
Would this be more indicative?